app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -39,9 +39,7 @@
                       readonly
                       v-on="on"
                       dense
-                      flat
```

Comment:
This works a LOT better and is simpler than what I did in my PR!
We will have to remove my style and use this approach. 👍

Reply:
I've added the two new style tweaks you mentioned in the comment above.
- links in my forms table
- formio button style